plugin/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -24,4 +24,10 @@ grant {
 
     // Circuit Breaker
     permission java.lang.RuntimePermission "getFileSystemAttributes";
+
+    // s3 client opens socket connections for to access repository
+    permission java.net.SocketPermission "*", "connect,resolve";
+
+    // aws credential file access
+    permission java.io.FilePermission "<<ALL FILES>>", "read";
```

Comment:
Why do we have to read `ALL FILES`?

Maybe something like:

```
permission java.io.FilePermission "/path/to/aws/credentials/*", "read";
```

Reply:
I tried different wild card DIRs for the credentials but they don't work. It has to be an exact match. Also we are not sure what is the exact file path in different machine/environment. Again, let's bring this to the security review and update as required.